style.css

```diff
@@ -0,0 +1,584 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Roboto+Slab:wght@500&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+}
+
+body::before {
+  display: block;
+  content: "";
+  height: 60px;
+}
+
+.header-section {
+  position: fixed;
+  background-color: white;
+  width: 100%;
+  top: 0;
+}
+
+.top-list-header {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 10px 90px 10px 0;
+  background-color: black;
+  color: white;
+  list-style: none;
+  gap: 1rem;
+  width: 100%;
+}
+
+.main-container {
+  margin: auto 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 30px 0 30px;
+  background-color: white;
+}
+
+.nav-list {
+  display: inline-flex;
+  justify-content: space-between;
+  list-style: none;
+}
+
+.campaign {
+  border: 3px solid red;
+  padding: 10px;
+}
+
+.logo a {
+  text-decoration: none;
+  cursor: pointer;
+  color: black;
+}
+
+.nav-item {
+  padding: 20px;
+  font-weight: 500;
+}
+
+.nav-item a {
+  text-decoration: none;
+  gap: 10px;
+  color: black;
+}
+
+.first-btn a {
+  color: #d3d3d3;
+  text-decoration: none;
+}
+
+.nav-item a:hover {
+  color: red;
+  transition: 0.5s;
+}
+
+.my-logo {
+  width: 15%;
+  height: 8%;
+  transition: transform 0.2s;
+}
+
+.my-logo:hover {
+  transform: scale(1.5);
+  -ms-transform: scale(1.5);
+  -webkit-transform: scale(1.5);
+}
+
+.title-section {
+  background-image: url(img/white_wave_bg.jpg);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  min-height: 60vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.greeting {
+  color: rgb(221, 56, 56);
+  font-size: 20px;
+  margin-bottom: 10px;
+}
+
+.intro {
+  color: rgb(184, 3, 3);
+  font-size: 20px;
+}
```

Comment:
- [ ] Kindly make sure you stuck the design using a `background-image`, `font-size`, `font-weight` on your text, let's take an example from this.

```` buch
  color: transparent;
  font-size: 2.05rem;
  font-weight: 1000;
  text-transform: uppercase;
  background-image: url(....image url....);
  background-position: -700px;
  -webkit-background-clip: text;

````
#### Templete
![Screenshot 2023-05-30 135309](https://github.com/Dorcas126/final-capstone-project-1/assets/57408419/6141f3d0-3d9f-4242-9661-c4de3a3a12d2)

#### Your design

![Screenshot 2023-05-30 142349](https://github.com/Dorcas126/final-capstone-project-1/assets/57408419/5b78e128-9eda-44c1-a1a2-c04c5407f43b)